### DISCUSSION TITLE
Setup MPD on a Freenas VM, no music on android phone

### DISCUSSION THREAD

**Discussion by Concubine** [MusicPlayerDaemon/MPD#1704] [Q&A]
Hey peeps, 

Great product, I love it!

So I setup my MPD on a freenas VM (Debian 11 4GIB/2Cores),  I mounted my NFS share to the default music folder and downloaded MALP, started testing on my LAN:

Had some output issues at the begining with ALSA but that is a given as the VM dosn't have a hardware audio output

Setup a streaming httpd output and while I seem to be able to control and play the songs I can only hear them on the stream output on the address of the HTTPD but not directly from the phone.

Here is my conf file:
[mpd.txt](https://github.com/MusicPlayerDaemon/MPD/files/10315766/mpd.txt)

Here is my log file:
[mpd.log](https://github.com/MusicPlayerDaemon/MPD/files/10315770/mpd.log)
(I know there are many write errors here, must be permission realted but I hardly think it is realted to the issue.)

Would any of you fine gentlepeeps care to help me understand why I can't hear anything from my configured android app?

EDIT:

So I read through the user guide and rearranged my conf file from the example, enabled winmm for windows output but am getting this error:

mpd[484]: Dec 28 19:41 : exception: Failed to configure output in line 297; No such audio output plugin: winmm

What plugin should I be using, what am I missing?

Music from android device still isn't working.

**Comment by MaxKellermann** [comment]
I don't understand the problem, and your description is too sparse on details. You said you can hear the stream output. Then you say you can't hear the stream output.

**Comment by MaxKellermann** [comment]
> Debian

> enabled winmm for windows output but am getting this error

Sigh. Don't you see that this doesn't make any sense? How can you expect a Windows-specific plugin to work on Linux? How did you even "enable" it?

Sorry, can't help you; even after my last reply, all you posted was this nonsense, no real information that would help anybody understand your problem.

**Comment by Concubine** [comment]
You know Sir you really are not nice.

I'm playing it on windows so I thought I needed to enable the windows
device for it to understand it needs to play on windows.

I've gone over other posts you answer, you have zero tolerance for people,
you don't try to understand the problem at all and think every one are
morons - I get some questions are dumb but not everyone can be in your
headspace.

I have an issue with the client I'm guessing but I'll find the solution on
my own, or just not use this daemon.

Thank you for a nasty experience.

**Comment by MaxKellermann** [comment]
Bye bye, you are now blocked here.
